data/_ui/tooltips.txt

```diff
@@ -1462,4 +1465,4 @@ tip "Scroll speed"
 	`The speed of scrolling in panels that can scroll.`
 
 tip "Date format"
-	`The format that dates in the game are displayed in.`
+	`The format that dates in the game are displayed in.`
```

Comment:
looks like you accidentally deleted the final LF

Reply:
May I ask what you mean by LF? Apperantly I cant find anything wrong there.

Reply:
there's a unix convention that every file should end with an empty line